fiaas_deploy_daemon/deployer/kubernetes/ready_check.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, app_spec, bookkeeper, lifecycle, lifecycle_subject, config):
         self._lifecycle_subject = lifecycle_subject
         self._fail_after_seconds = _calculate_fail_time(
             config.ready_check_timeout_multiplier,
-            app_spec.replicas,
+            app_spec.autoscaler.max_replicas,
```

Comment:
If we're going to set replicas=min in the no-HPA environments, then it will never reach max, so this will never report success. I think it should be min_replicas

Reply:
this is only to calculate the amount of time to wait I think? So setting min or max will only have an impact on how long we wait for all pods to be ready. If that's the case, changing to min_replicas may fail too early in the case where HPA scales to the max replicas and the difference between min and max is significant?

Reply:
Ah, yeah, you're right 👍